adapters/quick/src/main/scala/caliban/QuickRequestHandler.scala

```diff
@@ -25,6 +25,10 @@ final private class QuickRequestHandler[R](
   import QuickRequestHandler._
   import ValueJsoniter.stringListCodec
 
+  private val readerConfig: ReaderConfig = ReaderConfig
+    .withMaxBufSize(2147483645)
```

Comment:
Any reason to not use `Int.MaxValue`?

Reply:
From memory I think the max that it can be set to is `Int.MaxValue - 2`. Maybe we can use that instead though

Reply:
It's `Int.MaxValue - 2`, yeah. We can use that, if you'd prefer.

Reply:
Yeah I think that's a bit better